detection_rules/main.py

```diff
@@ -116,7 +116,11 @@ def import_rules_into_repo(input_file, required_only, directory):
     for contents in rule_contents:
         base_path = contents.get('name') or contents.get('rule', {}).get('name')
         base_path = rulename_to_filename(base_path) if base_path else base_path
-        rule_path = os.path.join(RULES_DIR, base_path) if base_path else None
+        rule_path = (
+            os.path.join(save_directory if save_directory is not None else RULES_DIRS[0], base_path)
+            if base_path
+            else None
```

Comment:
Do we need this? When would base_path be None since its generated from the rule name?

Reply:
Good point, will remove